shellbase-slack/src/main/scala/com/sumologic/shellbase/slack/PostCommandWithSlackThreads.scala

```diff
@@ -0,0 +1,41 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.sumologic.shellbase.slack
+
+trait PostCommandWithSlackThreads extends PostCommandToSlack {
+
+  override def postInformationToSlackThread(ts: String, timeDuration: Long, result: Boolean): Option[String] = {
+    val replyMessage = {
+      if (result) {
+        if (timeDuration >= 2) s"Hey @${username}， command success"
```

Comment:
We're tagging `username`, but not everyone's local username is the same as their slack name.  We should find a way to allow a user to fix this behavior.  It may be out of scope for this PR though.

Reply:
Yea... I was also concerning that, but seems we were tagging `username` in `sendSlackMessageIfConfigured` as well.

Reply:
That's for a whitelist/blacklist, so local username was ok.  Here we're going to use an invalid value, so we'll need some sort of improvement